api.planx.uk/saveAndReturn/sendEmail.ts

```diff
@@ -1,25 +1,35 @@
 import { NextFunction, Request, Response } from "express";
 import { sendSingleApplicationEmail, Template } from "./utils";
+import { sendSinglePaymentEmail } from "../inviteToPay";
 
 const sendSaveAndReturnEmail = async (
```

Comment:
It made sense to me to use the existing `/send-email/:template` endpoint defined in server.js, even though that calls this function - whose name has now fallen out of sync with what it does. But I failed to think of something better - any ideas? 

`routeEmailTemplate` maybe? It's not actually responsible for sending the email (and never has been), but rather it checks the provided payload and determines if we should then call a method to validate/send a Save&Return email or an Invite to Pay one.

Reply:
updated throughout to be `routeSendEmailRequest` :+1:

still lives within `saveAndReturn` subdirectory, but added file re-organisation to Trello debt ticket :broom: